nltk/test/tokenize.doctest

```diff
@@ -39,6 +39,9 @@ Some test strings.
     >>> s10 = "There were 300,000, but that wasn't enough."
     >>> word_tokenize(s10)
     ['There', 'were', '300,000', ',', 'but', 'that', 'was', "n't", 'enough', '.']
+    >>> s11 = "«Now that I can do.»"
```

Comment:
Possibly this will break the doctest in Python2. You might need to explicitly encode it as this instead:

```python
s11 = u'\xabNow that I can do.\xbb'
```